packages/components/src/Menu/Menu.css

```diff
@@ -97,7 +76,7 @@
 }
 
 .action:hover,
-.action:focus-visible {
+.action:focus {
```

Comment:
uncertain about this change and if it is correct. it was recently changed from focus -> focus-visible

the problem is that when the menu opens and the first element is programmatically focused by the `useFocusTrap` it was not visually focusing

with this style rule it is. not sure if maybe it should be both rules rather than just one or the other.

Reply:
`focus-visible` is intended to only show the focus "ring" when the user is operating the UI via keyboard/non-mouse input. I think in this particular case, the `background-color` should be present on both hover and focus, but we should move the outline styling to a separate `focus-visible` declaration.

Reply:
ok I've split that into its own declaration, but as far as I can tell it isn't doing anything. I've forced the `focus-visible` state and it's getting the styles, but even with an aggressive red colour - nada
![Components___Forms_and_Inputs___InputText___Web_-_Multiline_⋅_Storybook](https://github.com/GetJobber/atlantis/assets/11843215/cf75143c-6bff-4776-b7e7-26c74871e381)

Reply:
In Chrome/ium, I think you need to provide [full outline styles](https://developer.mozilla.org/en-US/docs/Web/CSS/outline) for anything to happen there, while Safari provides the style and size out of the box.

By setting the outline-color to transparent and supplying our `--shadow-focus` on `focus-visible`, we should get the results we want across browsers.

Reply:
ah gotcha. so if the styles were already trying to do that without the additional styles for Chrome - then it's currently not working and like you said, we can follow up on that later?

Reply:
Correct, probably easier to do in a more intentional/smaller followup